README.md

```diff
@@ -94,6 +94,15 @@ To create a new K3k cluster, use the following command:
 ```bash
 k3kcli cluster create mycluster
 ```
+> [!NOTE] Creating a K3k Cluster on a Rancher-Managed Host Cluster
```

Comment:
It looks like Github doesn't support this customization (yet?), sorry @jp-gouin :cry: 

Something like this should be ok though:

```suggestion
> [!NOTE]
> **Creating a K3k Cluster on a Rancher-Managed Host Cluster**
```